pandas/sparse/series.py

```diff
@@ -570,27 +570,13 @@ def copy(self, deep=True):
         return self._constructor(new_data, sparse_index=self.sp_index,
                                  fill_value=self.fill_value).__finalize__(self)
 
+    @Appender(generic._shared_docs['reindex'] % _shared_doc_kwargs)
     def reindex(self, index=None, method=None, copy=True, limit=None,
                 **kwargs):
-        """
-        Conform SparseSeries to new Index
-
-        See Series.reindex docstring for general behavior
 
-        Returns
-        -------
-        reindexed : SparseSeries
-        """
```

Comment:
so I think we need a check  additional kwargs are not pass (IOW, ``tolerance``, ``level``, ``fill_value``). I am actually not sure what happens if you pass these (prob ignored)?

so need some tests for these (should prob just raise ``ValueError`` as not supported by ``SparseSeries``)

further need to have somethin in the doc-string that show them (for ``Series``) or NOT for ``SparseSeries``.

so you need a substitution that can handle this.